rpcore/native/__init__.py

```diff
@@ -37,20 +37,20 @@
 
 # Store a global flag, indicating whether the C++ modules were loaded or the python
 # implemetation of them
-NATIVE_CXX_LOADED = False
+NATIVE_CXX_LOADED = True
```

Comment:
So I guess this one is here since the native modules are already built in panda3d, right?

Reply:
Yes. It would be better to switch to rplight module from panda3d and remove native and pynative modules from RP. It will keep RP's WHL python-only.